### PR TITLE
Implement sentiment score plot

### PR DESCRIPTION
**Pull request by ranjitprakash1986**

### CODE REVIEW

**Comment by ranjitprakash1986** [comment]
Withdrawing this pull request due to new updates

**Comment by ranjitprakash1986** [comment]
1. Added updates to the sentiment_score_plot function.
2. Added tests_tweets.csv under tests folder for testing.
3. Wrote related tests that need to be refined with proper testing.

File: src/pysentimentanalyzer/sentiment_score_plot.py [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+
+    Provides the most frequent sentiment score and the corresponding percentage
 
     Parameters
     ----------
     df : Pandas DataFrame
         DataFrame containing the reviews
     col: String
-        Column name in the dataframe that contains the reviews 
+        Name of the column that contains th reviews
```
Comment (erictsai1208) [inline src/pysentimentanalyzer/sentiment_score_plot.py]:
```suggestion
        Name of the column that contains the reviews
```

File: src/pysentimentanalyzer/sentiment_score_plot.py [hunk 2]
```diff
@@ -17,7 +26,38 @@ def sentiment_score_plot(df, col):
...
     >>> plt.show()
     """
-    pass
\ No newline at end of file
+
+    # Call the helper function to compute the sentiment and score
+    data = get_sentiment_and_score(df, col)
+
+    # Check that the column exists in dataframe
+    assert (
```
Comment (erictsai1208) [inline src/pysentimentanalyzer/sentiment_score_plot.py]: We should probably not use an `assert` here but rather an `if` statement to check if the column name is in the DataFrame.

**Review by erictsai1208 — CHANGES_REQUESTED** [review]
I feel like some small changes are needed. The last 2 tests do not really test the function. If you want to check that the user passes in the correct datatype, you should have checks in the `sentiment_score_plot` function.
e.g.
```
if not isinstance(df, pd.DataFrame):
    raise Exception("The first parameter should be a Pandas DataFrame.")
```
and then in the test you can check 
```
with pytest.raises(Exception):
    sentiment_score_plot("not a dataframe", "col_name")
```

File: src/pysentimentanalyzer/sentiment_score_plot.py (hunk 2 above)
Comment (ranjitprakash1986) [reply src/pysentimentanalyzer/sentiment_score_plot.py]: Hi Eric, can you please let me know the thought process behind it to understand why we wouldn't use an assert within a function. I assume you are requesting to use if-else and raise exception error. Then the exception should be detected in the pytests?
Any drawbacks of having an assert statement instead?

File: src/pysentimentanalyzer/sentiment_score_plot.py (hunk 1 above)
Comment (ranjitprakash1986) [reply src/pysentimentanalyzer/sentiment_score_plot.py]: I have made this change and pushed the updated commit

File: src/pysentimentanalyzer/sentiment_score_plot.py (hunk 2 above)
Comment (ranjitprakash1986) [reply src/pysentimentanalyzer/sentiment_score_plot.py]: I have made this change and pushed the updated commit

**Review by erictsai1208 — APPROVED** [review]
Looks good to merge

**Pull request merged**